src/framework/varQuery.mli

```diff
@@ -0,0 +1,6 @@
+type t =
+    Global of Cil.varinfo
+  | Node of { node : Node.t; fundec : Cil.fundec option; }
+val compare : t -> t -> int
+type 'v f = 'v -> unit
+val varqueries_from_names : Cil.file -> string list -> t list * string list
```

Comment:
I think now that there's a `.mli` file for this module, the documentation comments `(** ... *)` should be moved from the implementation to the interface, otherwise they aren't displayed as documentation anywhere.